### PR TITLE
167 pre release: tidy description and help files

### PR DESCRIPTION
**Pull request by jdhoffa**
Relates to all @jdhoffa tags in #167 
- [x] Review that help files are informative

### CODE REVIEW

File: R/match_name.R [hunk 1]
```diff
@@ -3,18 +3,20 @@
...
+#'
 #' @template alias-assign
 #' @template ignores-but-preserves-existing-groups
 #'
 #' @param loanbook,ald Dataframes structured like [r2dii.data::loanbook_demo]
 #'   and [r2dii.data::ald_demo].
-#' @param by_sector Should the combinations be done by sector?
-#' @param min_score A number (length-1) to set the minimum `score` values you
-#'   want to pick.
+#' @param by_sector Should names only be compared if entities belong to the same
```
Comment (maurolepore) [inline R/match_name.R]: I don't understand the word "entitles" in this context. Is this a correct use?

**Comment by maurolepore** [comment]
/document

@jdhoffa, not sure I mentioned that you can also document and style the code from here, by starting a comment with /document or /style. 

The magic comes from 
https://github.com/2DegreesInvesting/r2dii.match/blob/master/.github/workflows/pr-commands.yaml

See https://github.com/2DegreesInvesting/resources/issues/90

File: R/match_name.R (hunk 1 above)
Comment (jdhoffa) [reply R/match_name.R]: **entities, not entitles. I will update it to companies to make it more clear

**Comment by jdhoffa** [comment]
Well that's pretty fuckin cool!

File: R/match_name.R (hunk 1 above)
Comment (maurolepore) [reply R/match_name.R]: Sorry, my bad -- I didn't read carefully enougth. Entities makes sense and is clear. But up to you.

**Pull request merged**